tests/ssl-opt.sh

```diff
@@ -403,7 +403,6 @@ detect_required_features() {
 
     case "$CMD_LINE" in
```

Comment:
Note that there's a bug in the case for RSA below: it matches on `server2`, which includes `ssl_server2`. So there are probably test cases that we don't run in builds without RSA, but should.

We should change the expressions to something like `*/server2[-._]*` and so on, to only match certificate file names.